requirements/requirements.txt

```diff
@@ -1,3 +1,4 @@
+einops
```

Comment:
I believe this would probably be better suited to the requirements-dev file

Reply:
Done.